### PR TITLE
WIP: Overall docs improvements

### PR DESCRIPTION
**Pull request by drgomesp**
WIP

### CODE REVIEW

File: docs/articles/fundamentals/network.md [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+the format of ADRs allows for collaboration through a revision system.
+
+## Node Types
+---
+
+### Motor
+
+Motor is Sonr's reference light client implementation, and will have all the properties of a
+typical [Tendermint Light Client](https://docs.tendermint.com/master/tendermint-core/light-client.html),
+and is the best option for quickly interacting with the Sonr network. Some of its capabilities are:
```
Comment (joshLong145) [inline docs/articles/fundamentals/network.md:37]: The wording of this section is a bit flawed. we start off by mentioning our blockchain layer. then transition onto functionalities of the motor. Perhaps we should think about the flow of this file as there is more to the stack than what is being outlined  in the  `Sonr Network` section. We should describe Sonr as a the Sum of our two node types `Motor` and `Highway` which themselves leverage our Blockchain layer.

**Review by joshLong145 — CHANGES_REQUESTED** [review]
Left some comments.

File: docs/articles/contributing/getting-started.md [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+
+  If you have Go installed, [installing Task](https://taskfile.dev/#/installation) is as easy as running:
+ 
+- [Ignite CLI](https://docs.ignite.com/), a CLI tool for managing Cosmos-based blockchains.
+
+```bash
+go install github.com/go-task/task/v3/cmd/task@latest
+```
+
+
```
Comment (ntindle) [inline docs/articles/contributing/getting-started.md:26]: imo add it

File: docs/articles/contributing/guidelines.md [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+
+## Coding Principles
+
+## Coding Style
+
+## Branching and Committing
+
+## Pull Requests
+
+
```
Comment (ntindle) [inline docs/articles/contributing/guidelines.md:17]: imo keep it

File: docs/articles/fundamentals/network.md (hunk 1 above)
Comment (joshLong145) [reply docs/articles/fundamentals/network.md:37]: deleted my other comments as we can hide the pages for now. once we fix this wording up a little  and update i think its good to merge

**Comment by joshLong145** [comment]
Removed previous comments as they where no longer applicable after talking to others on those pages.

File: docs/sidebars.js [hunk 4]
```diff
@@ -16,6 +16,13 @@ const sidebars = {
   // But you can create a sidebar manually
   basicsSidebar: [
     'introduction',
+    {
+      type: 'category',
+      label: 'Fundamentals',
+      items: ['fundamentals/network', 'fundamentals/concepts'],
```
Comment (ntindle) [inline docs/sidebars.js:22]: Don’t link to unfinished pages or make it clear they are unfinished

**Pull request closed without merging**